src/redis/BetterSocialCronQueue.js

```diff
@@ -22,7 +22,7 @@ const {EVENT_DAILY_PROCESS_TRIGGER} = require('../processes/scoring-constant');
 /**
  * @typedef {Object} BetterSocialCronQueueInjection
  * @property {Bull.Queue} credderScoreQueue
- * @property {Bull.Queue} deleteActivityProcessQueue
+ * @property {Bull.Queue} removeActivityQueue
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `deleteActivityProcessQueue` to `removeActivityQueue` seems fine as long as the new queue name is consistently used throughout the codebase. Ensure that all references to the old queue name have been updated to avoid any undefined errors.
```diff
- * @property {Bull.Queue} deleteActivityProcessQueue
+ * @property {Bull.Queue} removeActivityQueue
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->